src/ert/run_models/base_run_model.py

```diff
@@ -777,7 +775,6 @@ def _evaluate_and_postprocess(
             env_pr_fm_step=self._env_pr_fm_step,
             forward_model_steps=self._forward_model_steps,
             substitutions=self._substitutions,
-            templates=self._templates,
```

Comment:
Think you can remove: `self._templates`, and by extension, also remove it from the constructor. Would of course have to be added to the constructor of the base run models that need it from the config, but think we should be explicit about it.

Reply:
Need to think about that, whether it is doable.